docs/specs/instrumentation.md

```diff
@@ -127,7 +127,7 @@ not result in data loss if the retry succeeds.**
 - Metrics
   - MUST increment the `<namespace>_discarded_events_total` counter by the
     number of events discarded.
-  - MUST include the listed properties as tags except the `reason` property.
+  - MUST NOT include `count` and `reason` from the `reason` property as tags.
```

Comment:
I'm not sure the new wording is an improvement. Do we want to make the tags optional with this change?

Reply:
It might not be needed at all, I noticed it when I was confirming all of properties were included as `tags` and was about to add `count` because 💭 - if it's less clear with a change, happy to close this instead.